view/theme/frio/style.php

```diff
@@ -93,7 +93,14 @@
 		$schemecssfile = 'view/theme/frio/scheme/default.css';
 	}
 }
-
+//If no scheme was set default to login theme.
+$nav_bg           = (empty($nav_bg)           ? Config::get('frio', 'nav_bg')           : $nav_bg);
```

Comment:
I think you have to move this block inside the `if ($a->module !== 'install') {` or the Config call will systematically fail.

Alternatively, you can integrate the below defaults in your code:

```
$nav_bg           = (empty($nav_bg) ? Config::get('frio', 'nav_bg', '#708fa0') : $nav_bg);
...
```

Even better: You can start this file by pulling the `Config::load('frio');` block at the top-level, add the hard-coded default to the `Config::get` calls like above, and then let the variables eventually be overriden by the `PConfig::get` calls. Saves lines instead of adding any.

Reply:
Tomorrow. Pretty tired right now. Don't merge yet. While tested, higher chance of breaking the network again because of a syntax error.

Ps. The config call didn't fail 😉